pkg/sfu/forwarder.go

```diff
@@ -1689,7 +1690,7 @@ func (f *Forwarder) processSourceSwitch(extPkt *buffer.ExtPacket, layer int32) e
 		// timestamp should be used as things will catch up to real time when channel capacity
 		// increases and pacer starts sending at faster rate.
 		//
-		// But, the challenege is distinguishing between the two cases. As a compromise, the difference
```

Comment:
This is the section I was trying to simplify by just using the expected time stamp. I need to spend more time thinking about this and simplify it.